### ISSUE TITLE
Installation fails due to conflicting urllib3 version

### ISSUE THREAD

**Issue by NeolithEra** [lubosson/webinfo-crawler#9]
Hi, users are unable to run **_webinfo-crawler_** due to dependency conflict with _**urllib3**_ package. 
As shown in the following full dependency graph of **_webinfo-crawler_**, webinfo-crawler requires _**urllib3==1.25.2**_，while **_requests==2.21.0_** requires _**urllib3>=1.21.1,<1.25**_.

According to pip’s _“first found wins”_ installation strategy, _**urllib3==1.25.2**_ is the actually installed version. 
However, _**urllib3==1.25.2**_ does not satisfy _**urllib3>=1.21.1,<1.25**_.

### Dependency tree
```
webinfo-crawler-master
| +-dbutils(version range:==1.3)
| +-pymysql(version range:==0.9.3)
| +-redis(version range:==3.2.0)
| +-requests(version range:==2.21.0)
| | +-certifi(version range:>=2017.4.17)
| | +-chardet(version range:<3.1.0,>=3.0.2)
| | +-idna(version range:>=2.5,<2.9)
| | +-urllib3(version range:>=1.21.1,<1.25)
| +-urllib3(version range:==1.25.2)
```

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Solution

1.	Fix your direct dependency to be urllib3>=1.21.1,<1.25. 
2.	Fix your direct dependency to be requests==2.22.0. 
3.	Remove your direct dependency urllib3, and use urllib3 transitively introduced by requests.

I have checked the above revisions will not affect your downstream projects now.

Which solution do you prefer, 1, 2 or 3?
@lubosson  Please let me know your choice. I can submit a PR to solve this issue.

**Comment by bouxin** [comment]
@NeolithEra I just upgraded requests to v2.22.0, the latest requests v2.22.0 officially supports urllib3 v1.25.2. solving it with command:
`pip3 install --upgrade requests==2.22.0`
I reran the `pip3 freeze > requirements.txt`, I will push it up right now, Thanks for recommending, the solution really meaningful, that's awesome!
